GDJS/Runtime/pixi-renderers/pixi-image-manager.ts

```diff
@@ -314,6 +315,11 @@ namespace gdjs {
       let loadedCount = 0;
       await Promise.all(
         [...this._resources.values()].map(async (resource) => {
+          if (resource.kind === 'video') {
+            // We do not preload videos, otherwise we don't have control on autoplay,
+            // and they would load automatically.
```

Comment:
It's not really the reason. We could absolutely preload videos and pass autoPlay to false. We can because we do it in the other method, and we know the kind of the resource (that's exactly what your `if` is doing).
We skip them because we don't want to preload them (as we did in the past).

```suggestion
            // We do not preload videos - we could (and ensure autoPlay is set to false) but it
            // seems not required and videos are streamed anyway so preloading is probably useless.
```

Reply:
I don't 100% agree with you. But your comm is fine
Doing `PIXI.Assets.load(videoResource)` doesn't allow passing `autoPlay: false` which triggers a play directly.
We could though check if it's a video, then use `PIXI.Texture.from()` (which allows that autoPlay param)